src/OrchardCore/OrchardCore.Liquid.Abstractions/ILiquidValueProvider.cs

```diff
@@ -0,0 +1,10 @@
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace OrchardCore.Liquid
+{
+    public interface ILiquidValueProvider
```

Comment:
If you agree with the change above, then we can just call it `ILiquidTemplateEventHandler` with a `RenderingAsync` method taking a `TemplateContext`

Reply:
Ok, i will do it.